css-text/Overview.bs

```diff
@@ -1296,7 +1299,7 @@ Line Breaking Details</h3>
 
 <pre class='propdef'>
 Name: overflow-wrap, word-wrap
-Value: normal | break-word
+Value: normal | break-word | break-spaces
```

Comment:
I don't think the resolution (SF or Syndey) specified if it was level 3 or 4. My main reason for preferring having it in L3 is:

* This value is new, but the behavior is already present in some browsers, triggered by a different property/value: Firefox and Safari wrap spaces like this when word-break is break-all (FF all preserved spaces, Safari only nbsp) (see http://output.jsbin.com/degaqe). Since we have clarified that the break-word property is not supposed to do that and that this is supposed to be triggered by overflow-wrap:break-spaces, rather than have them remove this behavior now and put it back later under a different name, I thought it made sense to add the switch in the current Level.

Also from a spec editing point of view, L3 is easier (but I agree this is not an important reason):

* L4 does not yet have the equivalent of section 4 (#white-space-processing), and that this makes some modification to it (section 4.1.3 point 4).